src/CommunityVoices/Public/embeddable.php

```diff
@@ -0,0 +1,165 @@
+          <div style="text-align:center; font-weight:bold; color:white">Natural Oberlin</div>
+        </div>
+        <div style="display: flex; flex-direction: column; width: 130px" class="carousel-selection-flex-item">
+          <div style="display: flex; justify-content: center; align-content: center; background-color: #b92d5d; border-radius: 10px; height: 105px; width: 130px"><img data-cc="6" onclick="setCategory(this)" src="/community-voices/uploads/4713" class="selector-img" style="cursor: pointer; margin: auto; max-width: 130px; max-height: 95px"></div>
+          <div style="text-align:center; font-weight:bold; color:white">Neighbors</div>
+        </div>
+        <div style="display: flex; flex-direction: column; width: 130px" class="carousel-selection-flex-item">
+          <div style="display: flex; justify-content: center; align-content: center; background-color: #002e7a; border-radius: 10px; height: 105px; width: 130px"><img data-cc="3" onclick="setCategory(this)" src="/community-voices/uploads/4714" class="selector-img" style="cursor: pointer; margin: auto; max-width: 130px; max-height: 95px"></div>
+          <div style="text-align:center; font-weight:bold; color:white">Next Generation</div>
+        </div>
+        <div style="display: flex; flex-direction: column; width: 130px" class="carousel-selection-flex-item">
+          <div style="display: flex; justify-content: center; align-content: center; background-color: #5c5c5c; border-radius: 10px; height: 105px; width: 130px"><img data-cc="2" onclick="setCategory(this)" src="/community-voices/uploads/4710" class="selector-img" style="cursor: pointer; margin: auto; max-width: 130px; max-height: 95px"></div>
+          <div style="text-align:center; font-weight:bold; color:white">Our Downtown</div>
+        </div>
+        <div style="display: flex; flex-direction: column; width: 130px" class="carousel-selection-flex-item">
+          <div style="display: flex; justify-content: center; align-content: center; background-color: #965117; border-radius: 10px; height: 105px; width: 130px"><img data-cc="1" onclick="setCategory(this)" src="/community-voices/uploads/4715" class="selector-img" style="cursor: pointer; margin: auto; max-width: 130px; max-height: 95px"></div>
+          <div style="text-align:center; font-weight:bold; color:white">Serving Our Community</div>
+        </div>
+      </div>
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+    <script>
+    var paths = <?php echo json_encode($files); ?>;
+    function setCategory(category) {
+      window.location.search = `content_category[]=${category.dataset.cc}`
```

Comment:
But does this reload the content on the page?

Reply:
Yes.

Reply:
Ahhh, it does a refresh.  I thought John / Rowan were envisioning a live update.